crates/debugger/src/tui/context.rs

```diff
@@ -3,14 +3,17 @@
 use crate::{Debugger, ExitReason};
 use alloy_primitives::Address;
 use crossterm::event::{Event, KeyCode, KeyEvent, KeyModifiers, MouseEvent, MouseEventKind};
-use eyre::Result;
-use foundry_evm_core::{debug::DebugStep, utils::CallKind};
-use std::ops::ControlFlow;
+use foundry_evm_core::{
+    debug::{DebugNodeFlat, DebugStep},
+    utils::CallKind,
+};
+use std::{cell::RefCell, ops::ControlFlow};
 
 /// This is currently used to remember last scroll position so screen doesn't wiggle as much.
 #[derive(Default)]
 pub(crate) struct DrawMemory {
-    pub(crate) current_startline: usize,
+    // TODO
```

Comment:
what's this todo for?

Reply:
It was the only field that gets modified while drawing, I don't know if removing the mutation breaks the debugger since I have no idea what that code does :D
I'll address in follow-up